scripts/docker-dev/geoserver/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# Provides this geoserver image with 'localhost' remapped to the host machine so db requests are made to host port 5432.
+from winsent/geoserver:2.9.2
```

Comment:
Prefer if you don't merge if you'd like changes; easily slips through unnoticed.